client/role/role.controller.js

```diff
@@ -0,0 +1,78 @@
+/* global angular */
+/* global _ */
+/* jshint node: true */
+'use strict';
+
+angular.module('app.role')
+  .controller('RoleCtrl', ['$scope', '$location', 'userService','alertService', 'user', 'adminLevels', 'locations', 'facilities', 'programs', 'facilityPrograms',  function($scope, $location, userService, alertService, user, adminLevels, locations, facilities, programs, facilityPrograms){
+
```

Comment:
Start considering the use of `var vm = this;` and `controllerAs` to void the direct use of `$scope`